programs/cleos/main.cpp

```diff
@@ -606,6 +606,137 @@ void print_return_value( const fc::variant& at ) {
    }
 }
 
+constexpr uint32_t receiver_width = 14;
+constexpr uint32_t function_width = 30;
```

Comment:
I doubt it matters but this was 28 before.

Reply:
Because of potential long names, I have made them all 30.